modules/markup/markdown/markdown_test.go

```diff
@@ -520,3 +520,40 @@ func TestMathBlock(t *testing.T) {
 
 	}
 }
+
+func TestTaskList(t *testing.T) {
+	testcases := []struct {
+		testcase string
+		expected string
+	}{
+		{
+			// data-source-position should take into account YAML frontmatter.
+			`---
+foo: bar
+---
+- [ ] task 1`,
+			`<details><summary><i class="icon table"></i></summary><table>
+<thead>
```

Comment:
Maybe there could be a `toc: false` in the meta section, to disable the `<details><summary>` part, then the output could be much simpler and clearer.

Reply:
You mean add `include_toc: false` to the YAML?  I can't get this to affect the details/summary.